src/test/java/io/fabric8/vertx/maven/plugin/it/DebugIT.java

```diff
@@ -0,0 +1,169 @@
+
+        verifier.verifyTextInLog("Listening for transport dt_socket at address: 5005");
+    }
+
+//    @Test
+//    public void testRedeployOnJavaChangeWithCustomLauncher() throws Exception {
+//        File testDir = initProject("projects/redeploy-with-custom-launcher-it");
+//        assertThat(testDir).isDirectory();
+//
+//        initVerifier(testDir);
+//        prepareProject(testDir, verifier);
+//
+//        run(verifier);
+//
+//        String response = getHttpResponse();
+//        assertThat(response).isEqualTo("Bonjour vert.x");
+//
+//        // Touch the java source code (verticle)
+//        File source = new File(testDir, "src/main/java/demo/SimpleVerticle.java");
+//        String uuid = UUID.randomUUID().toString();
+//        filter(source, ImmutableMap.of("vert.x", uuid));
+//
+//        // Wait until we get "uuid"
+//        await().atMost(1, TimeUnit.MINUTES).until(() -> getHttpResponse().equalsIgnoreCase("Bonjour " + uuid));
+//
+//        // Touch the launcher class
+//        source = new File(testDir, "src/main/java/demo/Main.java");
+//        String uuid2 = UUID.randomUUID().toString();
+//        filter(source, ImmutableMap.of("Bonjour", uuid2));
+//
+//        // Wait until we get "uuid uuid"
+//        await().atMost(1, TimeUnit.MINUTES).until(() -> getHttpResponse().equalsIgnoreCase(uuid2 + " " + uuid));
+//    }
+//
+//    @Test
+//    public void testRedeployOnResourceChange() throws Exception {
+//        File testDir = initProject("projects/redeploy-on-resource-change-it");
+//        assertThat(testDir).isDirectory();
+//
+//        initVerifier(testDir);
+//        prepareProject(testDir, verifier);
+//
+//        run(verifier);
+//
+//        String response = getHttpResponse();
+//        assertThat(response).startsWith("Hello");
+//
+//        // Touch the java source code
+//        File source = new File(testDir, "src/main/resources/some-text.txt");
+//        String uuid = UUID.randomUUID().toString();
+//        filter(source, ImmutableMap.of("Hello", uuid));
+//
+//        // Wait until we get "uuid"
+//        await().atMost(1, TimeUnit.MINUTES).until(() -> getHttpResponse().startsWith(uuid));
+//    }
+//
+//    @Test
+//    public void testRedeployOnResourceChangeManagedBySomeOtherPlugin() throws Exception {
+//        File testDir = initProject("projects/redeploy-with-some-plugin-it");
+//        assertThat(testDir).isDirectory();
+//
+//        initVerifier(testDir);
+//        prepareProject(testDir, verifier);
+//
+//        run(verifier);
+//
+//        String response = getHttpResponse();
+//        assertThat(response).contains("color: #f938ab");
+//
+//        // Touch the java source code
+//        File source = new File(testDir, "src/main/less/style.less");
+//        filter(source, ImmutableMap.of("#f938ab;", "green;"));
+//
+//        // Wait until we get "uuid"
+//        await().atMost(1, TimeUnit.MINUTES).until(() -> getHttpResponse().contains("color: #008000;"));
+//    }
+
```

Comment:
its better to remove L91-L162, not sure whats your idea on having commented , if nots required we shall remove it